controllers/users/users.js

```diff
@@ -0,0 +1,15 @@
+const User = require("../../db_models/usersScheme");
+
+module.exports.getUserInfo = (req, res) => {
+  try {
+    User.find({
+      _id: req.query.userId,
+    }).then((result) => {
+      res.status(200).json(result);
+    });
+  } catch {
+    return res
+      .status(500)
+      .send({ message: "Something wrong. Please wait for debug..." });
+  }
+};
```

Comment:
Please, change the name of the file to avoid duplicate filenames

Use index.js inside `/users` dir to import it as `require("../users"`) instead of `require("../users/users")`